packages/server/graphql/types/Team.ts

```diff
@@ -327,9 +327,13 @@ const Team: GraphQLObjectType = new GraphQLObjectType<ITeam, GQLContext>({
         }
       },
       description: 'All the team members actively associated with the team',
-      async resolve({id: teamId}, args: any, {authToken, dataLoader}) {
+      async resolve({id: teamId, orgId}, args: any, {authToken, dataLoader}) {
+        const viewerId = getUserId(authToken)
         const {sortBy = 'preferredName'} = args as {sortBy?: 'preferredName'}
-        if (!isTeamMember(authToken, teamId) && !isSuperUser(authToken)) return []
+        const isBillingLeader = await isUserBillingLeader(viewerId, orgId, dataLoader)
+        const canViewAllMembers =
```

Comment:
+1 actually all the billingLeader needs to see is the team lead. Not a big deal, but we could add a separate `teamLeads` field. The billing lead can see all org members anyways, but maybe some team members should be kept private?

Reply:
We need to show the team member count too, so I think we need access to all team members here